# Array of projected points
std_msgs/Header header
ProjectedPoint[] points
